.github/integration_tests/compliant_repo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Integration test to ensure CI fails when a non-existent repository is added
+
+# set git credentials
+git config user.name dfinity
+git config user.email dfinity@dfinity.org
+
+# create new branch
+git checkout -b integration-test-2
+
+# add compliant repository to for testing
+echo "test-compliant-repository" >> open-repositories.txt
+sort open-repositories.txt -o open-repositories.txt
+git add open-repositories.txt
+git commit -m 'add compliant repository'
+
+# create pull request
+git push --set-upstream origin integration-test-2
```

Comment:
Will this work if there's `integration-test-2` branch already with commit from previous push?

Reply:
It always deletes the branch at the end of the run, so it will work. If two people are running integration tests at the same time it will result in a failure, in which case one will have to be restarted once the other one finishes. This should happen so rarely though, that I don't think it needs a work-around.